### PR TITLE
Open communications

### PR DESCRIPTION
**Pull request by D4nte**
First draft to give an idea. Please review the concept. Need to fix typos and links.

### CODE REVIEW

**Comment by D4nte** [comment]
pending one some liquid announcements

File: blog/2020-11-11-open-communications.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+### References
+ 
+- [1] comit-rs uses GPLv3 https://github.com/comit-network/comit-rs/blob/dev/LICENSE.md
+- [2] ambrosia uses Apache 2.0 https://github.com/comit-network/ambrosia/blob/master/LICENSE
+- [3] Contributor agreement removal PR: https://github.com/comit-network/comit-rs/pull/3344
+- [4] We used to document designs in GitHub: https://github.com/comit-network/spikes/
+- [5] The comit-dev mailing list, it took us that much time to realise that mailing lists are the most powerful collaborative tools :) https://lists.comit.network/mailman/listinfo/comit-dev
+- [6] #comit-dev matrix chat: https://matrix.to/#/!eDtJfYgJutkmKTvbOH:matrix.org?via=matrix.org
+- [7] #comit-liquid TODO
```
Comment (bonomat) [inline blog/2020-11-11-open-communications.md:38]: Already created but no one in it yet. 
```suggestion
- [7] #comit-liquid chat: https://matrix.to/#/!VXJqVoUrHanVlQFVEU:matrix.org?via=matrix.org
```

File: blog/2020-11-11-open-communications.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### References
+ 
+- [1] comit-rs uses GPLv3 https://github.com/comit-network/comit-rs/blob/dev/LICENSE.md
+- [2] ambrosia uses Apache 2.0 https://github.com/comit-network/ambrosia/blob/master/LICENSE
+- [3] Contributor agreement removal PR: https://github.com/comit-network/comit-rs/pull/3344
+- [4] We used to document designs in GitHub: https://github.com/comit-network/spikes/
+- [5] The comit-dev mailing list, it took us that much time to realise that mailing lists are the most powerful collaborative tools :) https://lists.comit.network/mailman/listinfo/comit-dev
+- [6] #comit-dev matrix chat: https://matrix.to/#/!eDtJfYgJutkmKTvbOH:matrix.org?via=matrix.org
+- [7] #comit-liquid chat: https://matrix.to/#/!VXJqVoUrHanVlQFVEU:matrix.org?via=matrix.org; #comit-monero chat: https://matrix.to/#/!QqYPpVbtwYxRItuYiA:matrix.org?via=matrix.org
```
Comment (bonomat) [inline blog/2020-11-11-open-communications.md:39]: Is `comit-monero` on purpose not an extra item?

File: blog/2020-11-11-open-communications.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+At COMIT, we believe in an open financial system - truly inclusive, censorship resistant and non-discriminating.
+
+We knew from day one that opensource software was the only way to achieve our vision, and we love opensource so it never was an issue.
+
+However, we realised that while we licensed our project with open [[1]](https://github.com/comit-network/comit-rs/blob/dev/LICENSE.md) and very open licenses [[2]](https://github.com/comit-network/ambrosia/blob/master/LICENSE), there were other obstacles on the way to declare us truly open:
+
+- Our contributor agreement was restrictive, it is now gone [[3]](https://github.com/comit-network/comit-rs/pull/3344);
+- Our design discussions moved from open GitHub [[4]](https://github.com/comit-network/spikes/) to closed basecamp, we are now back in the open with the comit-dev mail list [[5]](https://lists.comit.network/mailman/listinfo/comit-dev);
+- We had private chats, we are now discussing in public on our dev [[6]](https://matrix.to/#/!eDtJfYgJutkmKTvbOH:matrix.org?via=matrix.org) and project chats [7] on matrix;
```
Comment (da-kami) [inline blog/2020-11-11-open-communications.md:19]: Did you not link to `[7]` on purpose? (because its 2 links?)

**Review by da-kami — APPROVED** [review]
Thanks for summing this up @D4nte !

File: blog/2020-11-11-open-communications.md (hunk 2 above)
Comment (D4nte) [reply blog/2020-11-11-open-communications.md:39]: Hum, I guess an extra item is better indeed.

File: blog/2020-11-11-open-communications.md (hunk 3 above)
Comment (D4nte) [reply blog/2020-11-11-open-communications.md:19]: It's related to @bonomat's comment. now fixed.

File: blog/2020-11-11-open-communications.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+tags: [vision,opensource,comit]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+At COMIT, we believe in an open financial system - truly inclusive, censorship resistant and non-discriminating.
+
+We knew from day one that opensource software was the only way to achieve our vision, and we love opensource so it never was an issue.
+
+However, we realised that while we licensed our project with open [[1]](https://github.com/comit-network/comit-rs/blob/dev/LICENSE.md) and very open licenses [[2]](https://github.com/comit-network/ambrosia/blob/master/LICENSE), there were other obstacles on the way to declare us truly open:
```
Comment (thomaseizinger) [inline blog/2020-11-11-open-communications.md:15]: "with open and very open"?

**Review by thomaseizinger — APPROVED** [review]

**Review by bonomat — APPROVED** [review]
I think this reads great 👏👏

**Comment by D4nte** [comment]
Will merge once CI is passed.

**Pull request merged**